src/pages/Home.tsx

```diff
@@ -1,25 +1,45 @@
-import React from 'react';
-import TwitteForm from '../features/twitter/TweetForm';
-import TweetsList from '../features/twitter/TweetsList';
-import EmptyStateMessage from '../features/twitter/emptyStateMessage/EmptyStateMessage';
-import { PageContainer } from '../styles/components';
-import { useRecoilState } from 'recoil';
-import { deleteConfirmDialogState, tweetState } from '../features/twitter/atoms/tweetAtom';
-import DeleteConfirmationDialog from '../features/twitter/deleteConfirmationDialog/DeleteConfirmationDialog';
+import React, { useState } from 'react';
+import {TwitteForm} from 'features/twitter/tweetForm/TweetForm';
+import {TweetsList} from 'features/twitter/tweetsList/TweetsList';
+import {EmptyStateMessage} from 'features/twitter/emptyStateMessage/EmptyStateMessage';
+import {DeleteConfirmationDialog} from 'features/twitter/deleteConfirmationDialog/DeleteConfirmationDialog';
+import {FocusFlag, Tweet} from 'models/Tweet';
 
-export default function Home() {
-    // User's name
-    const userName = 'Shlomi Masuri';
-
-    const [tweets] = useRecoilState(tweetState);
-    const [deleteConfirmDialog] = useRecoilState(deleteConfirmDialogState);
+export function Home() {
+    const [userName, setUserName] = useState('');
+    const [tweets, setTweets] = useState<Tweet[]>([]);
+    const [focus, setFocus] = useState<FocusFlag>({setFocus: false});
+    const [tweetIdToDelete, setTweetIdToDelete] = useState<string>('');
+    const onTweedAdd = (newTweet: Tweet) => {
+        setTweets([...tweets, newTweet]);
+    }
+    const onTweetIdSelectedForDelete = (tweetId: string) => {
+        setTweetIdToDelete(tweetId);
+    }
+    const handleDeleteConfirm =  (isConfirm: boolean) => {
+        if (isConfirm)
+            setTweets(tweets.filter(t => t.id!== tweetIdToDelete));
+        setTweetIdToDelete('');
+    }
+    const handleSetFocus = () => {
+        setFocus({setFocus: true});
+    }
 
     return (
-        <PageContainer>
-            
-            {deleteConfirmDialog !== '' && <DeleteConfirmationDialog/>}
-            <TwitteForm userName={userName}/>
-            {tweets?.length === 0 ? <EmptyStateMessage /> : <TweetsList />}
-        </PageContainer>
+        <>
+            {tweetIdToDelete !== '' && 
+                <DeleteConfirmationDialog handleConfirm={handleDeleteConfirm}/>
+            }
+            <label className='user-name-label'>Author Name:
+                <input type='text' name='userName' autoComplete='given-name' placeholder='Enter User Name'
+                className='user-input' value={userName} onChange={(e) => setUserName(e.target.value)}/>
+            </label>
+            <div className={'page-container'}>
+                <TwitteForm handleSubmit={onTweedAdd} userName={userName} focus={focus}/>
```

Comment:
you are inconsistent, some handlers are `on<Event Name>` and some are `handle<Event Name>`